examples/provider/provider.tf

```diff
@@ -1,9 +0,0 @@
-provider "vmp" {
```

Comment:
This file and variables.tf are needed for the documentation generator tool, can you add them back?